.env.test

```diff
@@ -0,0 +1,17 @@
+# Flask server configuration.
+FLASK_APP=todo_app/app
+FLASK_ENV=test
+FLASK_SKIP_DOTENV=True
+
+# Change the following values for local development.
+SECRET_KEY=test_secret_key
+
+# TEST Trello API Details
+TRELLO_API_KEY=this_is_trello_api_key_123
+TRELLO_TOKEN=and_this_is_the_trello_token_56789
+BOARD_ID=5fb51906cc7d395b1b145b0e
+
+TODO_ITEM_ID=5fb9923734e1420b507bc75e
+COMPLETED_ITEM_ID=5fb9924c9da5330af79e095c
+IN_PROGTESS_ITEM_ID=5fb9923734e1420b507bc75e
+REMOVE_ITEM_ID=5fb9923734e1420b507bc75e
```

Comment:
This appears to be only used by the tests, so we could move this to just be a test variable rather than an environment variable

Reply:
These have now been made test variables